radix_sort/Java/radix_sort.java

```diff
@@ -0,0 +1,40 @@
+package com.javatutorial;
+  
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp75peg948/radix_sort/Java/radix_sort.java
+++ b/tmp/tmp75peg948/radix_sort/Java/radix_sort.java
@@ -1,5 +1,5 @@
 package com.javatutorial;
-  
+
 import java.util.ArrayList;
 import java.util.List;
   
```